jacs-model-rendering/src/main/java/org/janelia/rendering/utils/ImageUtils.java

```diff
@@ -67,6 +66,54 @@ private static class TiffROI {
         int endZ;
     }
 
+    private static class SeekableStreamWrapper extends SeekableStream {
+        private final InputStream sourceStream;
+        private final SeekableStream seekableDelegate;
+
+        private SeekableStreamWrapper(InputStream sourceStream) {
+            this.sourceStream = sourceStream;
+            if (sourceStream instanceof SeekableStream) {
+                seekableDelegate = (SeekableStream) sourceStream;
+            } else {
+                seekableDelegate = new MemoryCacheSeekableStream(sourceStream);
+            };
```

Comment:
this is the culprit - when we use memorycacheableseekablestream - this does not close the wrapped source. I am not sure if this is a bug in MemoryCacheSeekableStream but that is really old code so the fix is to close it ourselves.